great_expectations/expectations/metrics/column_pair_map_metrics/column_pair_values_in_set.py

```diff
@@ -61,6 +65,19 @@ def _pandas(cls, column_A, column_B, **kwargs):
 
         return pd.Series(results)
 
+    @column_pair_condition_partial(engine=SqlAlchemyExecutionEngine)
+    def _sqlalchemy(cls, column_A, column_B, **kwargs):
+        value_pairs_set = kwargs.get("value_pairs_set")
+
+        if value_pairs_set is None:
+            # vacuously true
+            return sa.case((column_A == column_B, True), else_=True)
```

Comment:
Wanted to double-check. Do we test for the `vacuously true` condition in `test_map_column_pairs_in_set_metric_sa()`?

Reply:
No, that is a good call-out. I will need to merge this to keep @alexsherstinsky rolling, but he could add it to the expectation PR.